src/user-actions.ts

```diff
@@ -0,0 +1,45 @@
+export default interface User {
+  id: string;
+  login: string;
+  password: string;
+  age: number;
+  isDeleted: boolean;
+}
+
+interface UserRes {
+  id: string;
+  login: string;
+  age: number;
+}
+
+export function getResUser(user: User): UserRes {
+  const userRes: UserRes = {
+    id: user.id,
+    login: user.login,
+    age: user.age,
+  };
+  return userRes;
+}
+
+export function getResUsers(users: User[]): UserRes[] {
+  return users.map(user => getResUser(user));
+}
+
+export function getAutoSuggestUsers(
+  users: User[],
+  loginSubstring: string,
+  limit?: number
+): User[] {
+  const sortedByLogin = users.sort((a, b) =>
+    a.login.toLowerCase() < b.login.toLowerCase()
```

Comment:
unreadable comparator. Shorter is not always better. Refactor with `if` statements

Reply:
fixed